File: BHOI_/2014/BHOI 2014/Roboti/grader.pas
Program grader;
uses roboti;
const
   inbuf_len  = 1 shl 23;
   outbuf_len = 1 shl 10;
var
  r, p, q, v, e, i : LongInt;
  energije : Array of LongInt;
  inbuf : array [1..inbuf_len] of Byte;
  outbuf : array [1..outbuf_len] of Byte;
begin
  settextbuf(input, inbuf);
  settextbuf(output, outbuf);
  readln(r, p, q, v, e);
  SetLength(energije, e * 3);
  for i := 0 to (e * 3) - 1 do
    read(energije[i]);
  writeln(BrojMinimalnihPotrosnji(r, p, q, v, e, energije));
end.

File: BHOI_/2014/BHOI 2014/Roboti/grader.cpp
#include <iostream>
#include <cassert>

using namespace std;

int BrojMinimalnihPotrosnji(int r, int p, int q, int v, int e, int* energije);

int main() {
  int r, p, q, v, e;
  cin >> r >> p >> q >> v >> e;
  int* energije = new int[e * 3];
  for (int i = 0; i < e * 3; i++)
      cin >> energije[i];
  cout << BrojMinimalnihPotrosnji(r, p, q, v, e, energije) << endl;
  return 0;
}
